### ISSUE TITLE
Check all GUI and help improvements for clarity

### ISSUE THREAD

**Issue by ljod** [SlicerDMRI/SlicerDMRI#51]
Sonia, please check and test the following improved things in SlicerDMRI before you come to Slicer day on Aug 30, 2016. Please share your per-module and overall feedback in a google document.

Improvements to check:
-- Overall module category reorganization for Diffusion.
-- Module changes: names, categories, GUI labels and descriptions (tooltips).
-- All module help pages.

Specific modules to be checked are listed below:
DiffusionTensorEstimation
DiffusionBrainMasking
DiffusionTensorScalarMaps
TractographyMeasurements
Tractography Seeding
Tractography ROI Seeding
Tractography ROI Selection
UKFTractography
Tractography To Mask Image
DiffusionWeightedDICOMImport (DWIConvert)

**Comment by ihnorton** [comment]
Notes from 20160916 lab meeting (cc @spujol)

Tensor Estimation
- [ ] move LS/WLS to advanced
  - [ ] check on WLS reliability and possibly make default (previously Teem issues?)

DT Scalar Maps
- [ ] "Output Volume" include name of scalar automatically
- [ ] Add spaces between words in the (displayed) measurement types?
- [ ] move FA -> first in list

Tractography Seeding / ROI Seeding
- [ ] unify measure selector (Linear Measure vs FA) and start/stop threshold sliders move -> seeding options
- [ ] "mm" displayed in Intx seeding vs "(mm)" in label in ROI (probably can't set measure units in CLI.
- [ ] Remove "Multiple File Output" option from CLI
- [ ] Intx Seeding w/ labelmap only works with "use Index Space"
- [ ] "Inter **C** tive" typo
- [ ] remove "Path" from "Min Path Length"
- [ ] ROI: Index Space, Spacing, "use Random grid" -> Advanced

**Comment by ihnorton** [comment]
Proposed GUI panel update in TractographySeeding
(branch https://github.com/ihnorton/SlicerDMRI/tree/intx_gui_update)

Labelmap                            |  Fiberbundle
:-------------------------:|:-------------------------:
! ![image](https://cloud.githubusercontent.com/assets/327706/18789415/ed6a1222-8178-11e6-8b55-b7ec46aef13d.png)  |  ![image](https://cloud.githubusercontent.com/assets/327706/18789432/02a7baa4-8179-11e6-9c04-67ae3b40ebb5.png)

**Comment by ljod** [comment]
Looks generally good. The labelmap one seems to have more inputs than a labelmap. Does that work?

I'm not sure what the exclamation point under labelmap or the parameters node under fiber bundle mean.

If FA is for starting and stopping, should it be called stopping criteria? Perhaps just "anisotropy measure" then "seeding threshold" and "stopping threshold"

I think integration step length should move to advanced.

**Comment by ihnorton** [comment]
> The labelmap one seems to have more inputs than a labelmap. Does that work?

Not sure I follow, the sliders mostly work the same, except for the start threshold.

I wonder if we should unify "Seed Spacing" and "Fiducial Seeding Step Size" and move that to Advanced?

**Comment by ljod** [comment]
the left one says select a markups fiducial but I thought it was the labelmap ROI seeding CLI. I thought that did not work with fiducials. I don't understand why the two modules have the same inputs. 

I think the left one, the ROIseeding CLI, should default to index space, i.e. one seed per voxel. Please confirm this is what index space means in the code. Increasing the amount of seeding may be commonly needed, though, so I am not sure if it should be in advanced.

I don't know what the fiducial seeding step size is. Is it the seed spacing or is it the tract step size?

also integration step length should be called "Step Size" instead, and the tooltip could incorporate the old name

**Comment by ljod** [comment]
Also, if seeding from a model I don't understand how the fiducial-related parameters apply. Is every model point seeded?

**Comment by ihnorton** [comment]
> the left one says select a markups fiducial but I thought it was the labelmap ROI seeding CLI. I thought that did not work with fiducials. I don't understand why the two modules have the same inputs.

It's one module, the middle (light gray background) panel changes contents depending on whether a labelmap or FB is selected. It says "Markups Fiducial" because that is the first node type that the selector is configured to accept, and I didn't have a labelmap selected.

Will double-check on the other ?s.

**Comment by ljod** [comment]
I still don't get it. I thought there were two different modules?

**Comment by ljod** [comment]
Oh I had thought this was about unifying the interface between the interactive and CLI modules, which I had thought was the main goal.

SO we work on this one first then unify the CLI information with it for clarity. OK.

A first pass changing the label names as I suggested above will help. 

Are you also trying to unify across inputs, so model vs fiducial vs labelmap? Is that the goal here?

**Comment by ihnorton** [comment]
> Are you also trying to unify across inputs, so model vs fiducial vs labelmap? Is that the goal here?

Yes. For as many common settings as possible -- moving to either "Tractography Parameters" or Advanced.

**Comment by ihnorton** [comment]
> Are you also trying to unify across inputs, so model vs fiducial vs labelmap? Is that the goal here?

I did some of the input label unifications in previous commits.
